PhysicsTools/NanoAOD/interface/SimpleFlatTableProducer.h

```diff
@@ -121,7 +121,7 @@ class ValueMapVariable : public ValueMapVariableBase<ObjType, TIn, ValType> {
                    edm::ConsumesCollector &&cc,
                    bool skipNonExistingSrc = false)
       : ValueMapVariableBase<ObjType, TIn, ValType>(aname, cfg, std::move(cc), skipNonExistingSrc) {}
-  ValType eval(const edm::Handle<edm::ValueMap<TIn>> &vmap, const edm::Ptr<ObjType> &op) const {
+  ValType eval(const edm::Handle<edm::ValueMap<TIn>> &vmap, const edm::Ptr<ObjType> &op) const override {
```

Comment:
was there anything failing without the `override` ?

Reply:
No, but it clearly indicates that this overrides method from base class; it is also consistent with what is done for fill method.